pennylane_lightning/core/src/simulators/lightning_qubit/bindings/LQubitBindings.hpp

```diff
@@ -183,6 +183,10 @@ void registerBackendClassSpecificBindings(PyClass &pyclass) {
         .def(py::init([](std::size_t num_qubits) {
             return new StateVectorT(num_qubits);
         }))
+        .def("updateNumQubits",
```

Comment:
We don't need this method --  should prefer the creation of new instances if there isn't a performance degradation involved with this.